akka-http-backend/src/main/scala/com/softwaremill/sttp/akkahttp/AkkaHttpBackend.scala

```diff
@@ -358,14 +385,15 @@ object AkkaHttpBackend {
       customHttpsContext: Option[HttpsConnectionContext] = None,
       customConnectionPoolSettings: Option[ConnectionPoolSettings] = None,
       customLog: Option[LoggingAdapter] = None
+  )(
+      http: AkkaHttpClient = AkkaHttpClient.fromAkkaHttpExt(actorSystem, customHttpsContext, customLog)
   )(implicit ec: ExecutionContext = ExecutionContext.Implicits.global): SttpBackend[Future, Source[ByteString, Any]] =
```

Comment:
To be honest I wanted to remove this default. Defaults don't play well with implicits (in terms of understandability) :)

Reply:
well, see above :)